core/pkg/opencost/opencost_codecs.go

```diff
@@ -457,6 +457,7 @@ func (target *Allocation) MarshalBinaryWithContext(ctx *EncodingContext) (err er
 	}
 	// --- [end][write][alias](LbAllocations) ---
 
+	buff.WriteFloat64(target.UnmountedPVCost) // write float64
```

Comment:
is this coming from Kubecost?

Reply:
No, this is being written from the struct definitions -- although I'm glad you spotted this. I think this field is supposed to be ignored. @ameijer Can I add the ignore to this changeset? Did I somehow stomp on your changes or are they still in-flight?